src/submission/migrations/0061_auto_20210923_1615.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-09-23 15:15
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def setup_display_emails(apps, schema_editor):
+    Article = apps.get_model("submission", "Article")
+
+    for article in Article.objects.all():
+        if article.correspondence_author:
+            article.frozenauthor_set.filter(
+                author__pk=article.correspondence_author.pk,
+            ).update(
+                display_email=True,
+            )
```

Comment:
for consistency with the current logic, we need to also set to `True` the first author for each of the articles